### PR TITLE
Add notes following kickstart livestream

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Document the things we did for the kickstart livestream.

This also moves the livestream content out of "competition", as we do more livestreams than just the competition event.

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
Does this help with https://github.com/srobo/tasks/issues/305 and/or https://github.com/srobo/tasks/issues/707?

**Review by WillB97 — APPROVED** [review]

**Comment by RealOrangeOne** [comment]
I think https://github.com/srobo/tasks/issues/707 was solved a while ago, but this definitely helps it. https://github.com/srobo/tasks/issues/305 is more aspirational.

File: docs/programme/livestream/scenes.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ This page documents a few of the core scenes which are used for livestreams. Not
...
 The various assets useful for creating these scenes can be found [here](https://drive.google.com/drive/folders/1pr1vKpnonxFoO8O5chsH5DF7Vsd0Tfih?usp=sharing).
 
+## Presentation
+
+Events such as Kickstart are predominantly a presentation.
+
+For the presentation, it's recommended to use a window-based capture, rather than screen-based, to reduce the risk of accidentally putting sensitive or unrelated content over the slides.
+
+Long presentations should still show the host. Choose a bottom corner of the screen which won't cover any content and keep their camera fixed there. Slides may need modifying to ensure important content isn't covered.
+
```
Comment (antonnikitin97) [inline docs/programme/livestream/scenes.md:14]: May be worth adding a section here on audio, namely not using global audio sources 
image
(the only exception _maybe_ being microphones as we'll have an interface to mute channels anyway) and individually adding sources in OBS. This means desktop audio won't come through on the stream. @ZachWilliamsSR last year showed me this tool (https://vb-audio.com/Voicemeeter/) to allow you to create "virtual" audio sources to pipe applications not via the main speaker. 

In Windows 11 in the mixer option you can also choose which output device to send audio to:
image

Comment (RealOrangeOne) [reply docs/programme/livestream/scenes.md:14]: Great idea, will add!

File: docs/programme/livestream/audio.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+As background music during livestreams, we'll often play quiet music.
+
+For finals events, more dramatic music is played vs other sessions, to emphasize the importance. For finals events, we have heavily used [Monstercat](https://www.monstercat.com/) in the past, and for other events [incompetech](https://incompetech.com/), as these sources can be used without purchasing a licence, provided the required attribution is given. We have also used [Epidemic Sound](https://www.epidemicsound.com/), however this is a paid-for service.
+
+There is no simple playlist used for each event. The playlists for each event are built off the previous, with any new finds added and disliked tracks removed.
+
+During live events, the music played throughout the venue needs to be stripped out, usually through a separate audio feed.
+
+Audio can either be played directly through OBS using the VLC source, or through a desktop audio player, so long as any other system sound effects are muted. When playing audio, be sure to [normalize it](https://www.alphr.com/normalize-volume-vlc) to ensure the volume doesn't vary.
```
Comment (antonnikitin97) [inline docs/programme/livestream/audio.md:20]: "so long as any other system sound effects are muted." 
- This is negated by using the virtual audio devices described above :) Maybe worth moving that up there as a "this is why we do it?"

**Comment by antonnikitin97** [comment]
🚢 it.

**Review by antonnikitin97 — APPROVED** [review]

**Pull request merged**